decidim-bulletin_board-app/app/views/sandbox/elections/index.html.erb

```diff
@@ -36,12 +36,13 @@
             <% elsif election.vote? %>
               <%= link_to "Vote", vote_sandbox_election_path(election) %> |
               <%= link_to "End vote", end_vote_sandbox_election_path(election) %>  |
+              <%= link_to "Load test stats", load_test_stats_sandbox_election_path(election), download: "load_test_stats.json" %>  |
               <button class="show-input-button"> Generate bulk votes </button>
               <div class="generate-votes-input-section">
                 <input class="generate-votes-input" type="number" placeholder="How many votes?" value="<%= default_bulk_votes_number %>"/>
                 <button class="generate-votes-button">Generate votes</button>
               </div>
-              <%= link_to "Download votes", download_bulk_votes_sandbox_election_path(election), class: "download-votes-button", download: 'bulk_votes.csv' %>
+              <%= link_to "Download votes", download_bulk_votes_sandbox_election_path(election), class: "download-votes-button", download: "bulk_votes.csv" %>
```

Comment:
Mmm, do we have a constant like `Sandbox::GenerateLoadTestStats::STATS_OUTPUT_FILE_PATH`for this?